PhysicsTools/PatAlgos/python/producersLayer1/jetProducer_cfi.py

```diff
@@ -47,7 +47,12 @@
         cms.InputTag("pfCombinedInclusiveSecondaryVertexV2BJetTags"),
         cms.InputTag("softPFMuonBJetTags"),
         cms.InputTag("softPFElectronBJetTags"),
-        cms.InputTag("pfCombinedMVABJetTags")
+        cms.InputTag("pfCombinedMVABJetTags"),
+        cms.InputTag("pfCombinedSecondaryVertexSoftLeptonBJetTags"),
```

Comment:
Was it discussed and agreed anywhere to add `pfCombinedSecondaryVertexSoftLeptonBJetTags` here? From the last performance studies I've seen, this tagger was not performing according to the expectation so unless that changed, I would rather leave out of the default set of taggers. Also, `pfCombinedMVABJetTags` now appears twice.